README.md

```diff
@@ -345,14 +345,15 @@ val forkedClient = defaultHttpClient.fork {
     A Request Logging Interceptor. 
     
     Parameters:
-    1.  `log: (String) -> Unit = ::println`:  function as a parameter to consume the log message. It defaults to `println`. Logs Request body when present.
+    1.  `outputCurlCommand: Boolean = false`:  if it's true, output curl command on log message.
+    2.  `log: (String) -> Unit = ::println`:  function as a parameter to consume the log message. It defaults to `println`. Logs Request body when present.
```

Comment:
Because,  `log` argument is function, this argument order cannot break code.
Below code is working correctly.
```
val client = defaultHttpClient.fork {
            interceptors {
                ///////old version code (outputCurlCommand argument does not exist)//////
                //no argument
                +LoggingInterceptor()
                //only log argument:
                +LoggingInterceptor { log ->
                    println(log)
                }
                ///////new version code (outputCurlCommand argument exists)//////
                //only outputCurlCommand
                +LoggingInterceptor(true)
                //outputCurlCommand and log arguments:
                +LoggingInterceptor(true) { log ->
                    println(log)
                }
            }
        }
```

I think `log` is function and need two or more line's code, so this order is better.
What do you think about it?

Reply:
You are definitely right, I was wrong.